{"uuid": "5d3fef9e-9265-47f8-a231-1f2c0fec85be", "children": ["2b09ec3f-5a8d-4430-b1ea-7ad8d1fa729b"], "befores": [{"name": "zs_login", "status": "failed", "statusDetails": {"message": "AssertionError: 前台登录失败\n", "trace": "  File \"D:\\ruanjian\\python\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"D:\\ruanjian\\python\\lib\\site-packages\\_pytest\\fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"D:\\ruanjian\\python\\lib\\site-packages\\_pytest\\fixtures.py\", line 932, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"D:\\daima\\xin-daima\\two-pytest-api-automation\\case\\conftest.py\", line 42, in zs_login\n    login_zs(zhangsan_info,'zhangsan_token')\n  File \"D:\\daima\\xin-daima\\two-pytest-api-automation\\common\\logintoken.py\", line 60, in login_zs\n    return Login(admin_info,zs_token)\n  File \"D:\\daima\\xin-daima\\two-pytest-api-automation\\common\\logintoken.py\", line 27, in __init__\n    assert result['code'] == 200,\"前台登录失败\"\n"}, "steps": [{"name": "admin账号登录", "status": "passed", "steps": [{"name": "响应日志", "status": "passed", "parameters": [{"name": "status_code", "value": "200"}, {"name": "text", "value": "{'code': 500, 'message': 'Bad credentials', 'data': None}"}], "start": 1636681171527, "stop": 1636681171527}, {"name": "请求日志", "status": "passed", "parameters": [{"name": "url", "value": "'http://121.37.169.128:8201/mall-member/sso/login?platform=test&time=1636681171473&password=123456&username=admin1'"}, {"name": "method", "value": "'post'"}, {"name": "body", "value": "{}"}, {"name": "headers", "value": "{'Content-Type': 'application/json'}"}], "start": 1636681171527, "stop": 1636681171527}], "start": 1636681171473, "stop": 1636681171527}], "start": 1636681171473, "stop": 1636681171527}], "start": 1636681171473, "stop": 1636681171589}